test_priorityq.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import priorityq
+
+
+def test_insert_peek(empty_pqueue):
+    """Test adding items to the list with correct priority and peeking."""
+    # tests that adding lower priority doesn't change first
+    empty_pqueue.insert(1, 1)
+    assert empty_pqueue.peek() == 1
+    empty_pqueue.insert(2, 2)
+    assert empty_pqueue.peek() == 1
+    empty_pqueue.insert(3, 3)
+    assert empty_pqueue.peek() == 1
+    # test add value with priority greater than number initial priority range
+    empty_pqueue.insert(4, 15)
+    assert empty_pqueue.peek() == 1
+    # tests that lower priorities were inserted correctly
```

Comment:
I would argue that the item with priority 15 was not inserted correctly, since it ended up in the queue with priority 3 items.  The behavior here is going to be quite unexpected for the user if you insert another priority three item.